playbooks/archivematica-src-trusty/docs/digital-ocean-install-example.rst

```diff
@@ -0,0 +1,138 @@
+How to Deploy Archivematica to a Digital Ocean Droplet
+================================================================================
+
+This document describes how to deploy Archivematica to a Digital Ocean droplet
+(i.e., virtual private server, VPS).  It assumes that you have basic
+proficiency with the Unix command-line and that you have the following
+installed.
+
+- git
+- Python
+- Ansible (http://docs.ansible.com/ansible/intro_installation.html)
+
+We are also assuming that you have a Digital Ocean account and that you have
+created a new droplet. The following URL may be useful for accomplishing this.
+
+- https://www.digitalocean.com/community/tutorials/how-to-create-your-first-digitalocean-droplet-virtual-server
+
+In this example, we are using Ubuntu 14.04.
+
+
+1. Clone the git repository that contains the Ansible configuration files which
+   will be used to install Archivematica and all of its dependencies onto the
+   Digital Ocean droplet::
+
+    $ git clone https://github.com/artefactual/deploy-pub.git
+
+2. Download the Ansible roles that will install Archivematica and its
+   dependencies::
+
+    $ cd deploy-pub/playbooks/archivematica
+    $ ansible-galaxy install -f -p roles/ -r requirements.yml
+
+3. Create a ``hosts`` file to tell Ansible the alias for our droplet (``am-do``),
+   its IP address and that we want to use the root user (where
+   ``xxx.xxx.xxx.xxx`` is the droplet's actual IP)::
+
+    $ cat hosts
+    am-do ansible_host=xxx.xxx.xxx.xxx ansible_user=root
+
+4. Modify the Ansible config file ``ansible.cfg`` to point to our ``hosts`` file::
+
+    $ cat ansible.cfg
+    [defaults]
+    nocows = 1
+    inventory = hosts
+
+5. If you do not have a SSH key, create one now (accepting the defaults)::
+
+    $ ssh-keygen -t rsa
+
+6. Copy the output of the above command to your clipboard and save it to your
+   digital ocean droplet in the "New SSH Key" web interface (see
+   https://cloud.digitalocean.com/settings/security)::
+
+    $ cat ~/.ssh/id_rsa.pub
+
+7. Use Ansible to create a new user on our Digital Ocean droplet. Create a file
+   (an Ansible playbook) called ``user.yml`` which has the content indicated by
+   the output of ``cat`` below::
```

Comment:
Why not create & commit the user.yml file, instead of including instructions to create it?  Or is it only needed for deploying to Digital Ocean and not other providers?